_test/CMakeLists.txt

```diff
@@ -36,8 +36,12 @@ foreach(_test_dir ${TEST_DIRECTORIES})
     # The MPI tests require that `herbert_on.m` be on the path for all workers
     # when Matlab starts up: add the path of the directory it's copied into
     # to the MATLABPATH environment variable when tests are run.
+    set(MATLABPATH "${MATLABPATH}:${CMAKE_BINARY_DIR}/herbert_config")
```

Comment:
this is done in every loop; the path variable can be constructed outside the loop and reused